static/app/components/searchSyntax/parser.tsx

```diff
@@ -600,9 +598,103 @@ export class TokenConverter {
       return this.checkInvalidInFilter(value as InFilter['value']);
     }
 
+    if (
+      [
+        FilterType.AggregateDate,
+        FilterType.AggregateDuration,
+        FilterType.AggregateNumeric,
+        FilterType.AggregatePercentage,
+        FilterType.AggregateRelativeDate,
+      ].includes(filter)
+    ) {
```

Comment:
```suggestion
    const isAggregateFilter = [
      FilterType.AggregateDate,
      FilterType.AggregateDuration,
      FilterType.AggregateNumeric,
      FilterType.AggregatePercentage,
      FilterType.AggregateRelativeDate,
    ].includes(filter)

    if (isAggregateFilter) {
```

Marginally easier to understand